api/src/main/java/org/openmrs/module/appointments/service/AppointmentsService.java

```diff
@@ -38,6 +38,9 @@ public interface AppointmentsService {
     @Authorized({VIEW_APPOINTMENTS, MANAGE_APPOINTMENTS})
     List<Appointment> getAllAppointmentsReminder(String hours);
 
+    @Transactional
+    @Authorized({VIEW_APPOINTMENTS, MANAGE_APPOINTMENTS})
+    List<Appointment> getAllAppointments(Date forDate, String status);
```

Comment:
See my earlier comment, this should take in an AppointmentStatus enum, not a String.

Reply:
Agreed!